### PR TITLE
PKG: separate building process from setup.py

### PR DESCRIPTION
**Pull request by ghost**
move the building process from setup.py to a separate file, make it an
executable
ship the apache configuration file if possible
adapt the debian build rules

### CODE REVIEW

**Comment by ghost** [comment]
@swilde could you please have a look at the debian-packaging-related changes? They look a bit hacky to me, but it seems they work.

File: setup.py [hunk 1]
```diff
@@ -1,65 +1,56 @@
...
     install_requires=[
         "intelmq-api",
+        "mako",
     ],
     include_package_data=True,
     url='https://github.com/certtools/intelmq-manager/',
     description=("IntelMQ Manager is a graphical interface to manage"
                  " configurations for the IntelMQ framework."),
-    data_files=data_files
+    data_files=[('/etc/intelmq', ['contrib/manager-apache.conf'])],  # required for deb packaing
```
Comment (monoidic) [inline setup.py]: Slight typo: packa**g**ing

**Review by monoidic — APPROVED** [review]
I'm not very familiar with Debian packaging, but everything LGTM

**Comment by ghost** [comment]
Thanks @monoidic !

@b1rger could you do me a favour and have a look at the Debian packaging part? I'm sure there's a better solution than my hack

**Comment by b1rger** [comment]
> @b1rger could you do me a favour and have a look at the Debian packaging part? I'm sure there's a better solution than my hack

LGTM (not sure if `override_dh_install` is the correct target, you could probably also use `override_dh_build` for the build step, but that's nitpicking)

**Comment by ghost** [comment]
Thanks @b1rger !

**Pull request merged**